src/main/java/com/squareup/javapoet/CodeBlock.java

```diff
@@ -269,6 +278,13 @@ public Builder addStatement(String format, Object... args) {
       return this;
     }
 
+    public Builder addStatement(CodeBlock codeBlock) {
```

Comment:
There's no indentation to worry about? It always came out correct, because it's a single line.